app/scripts/views/ready.js

```diff
@@ -64,15 +64,11 @@ function (Cocktail, BaseView, Template, Session, Xss, Strings,
         service: this.relier.get('service'),
         language: this.language,
         el: this.$('.marketing-area'),
-        metrics: this.metrics,
-        able: this._able
+        metrics: this.metrics
       };
 
-      // fallback in case experiment has not concluded
-      var fxIos = this._able.choose('fxIos') || {};
-
-      // Fx iOS scheduled to launch 6/2
-      if (fxIos.launched) {
+      // spring campaign scheduled to launch 6/2
+      if (this._able.choose('springCampaign2015')) {
```

Comment:
@jpetto - have you created a new PR in mozilla/fxa-content-experiments to enable this on June 2nd?